### PR TITLE
ui. added configuration fields in Settings page

### PR DESCRIPTION
**Pull request by edospadoni**
https://github.com/nethesis/ns8-nethvoice/issues/80

### CODE REVIEW

**Comment by edospadoni** [comment]
To test:
```bash
add-module ghcr.io/nethesis/nethvoice-proxy:ns8-nethvoice-80 1
```

File: ui/public/metadata.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
   ],
   "docs": {
     "documentation_url": "https://github.com/nethesis/ns8-nethvoice-proxy",
-    "code_url": "https://github.com/nethesis/ns8-nethvoice-proxy"
+    "code_url": "https://github.com/nethesis/ns8-nethvoice-proxy",
+    "bug_url": "https://github.com/nethesis/ns8-nethvoice/issues"
```
Comment (Amygos) [inline ui/public/metadata.json:16]: Why did you add the `bug_url` field?

**Comment by Amygos** [comment]
Can you add a screenshot of the UI?

File: ui/public/metadata.json (hunk 1 above)
Comment (edospadoni) [reply ui/public/metadata.json:16]: Because it was missing, in the UI there is a field that shows it but it was missing.

image

**Comment by edospadoni** [comment]
> Can you add a screenshot of the UI?

Yes

image


You can test it with: `add-module ghcr.io/nethesis/nethvoice-proxy:ns8-nethvoice-80 1`

File: ui/public/metadata.json (hunk 1 above)
Comment (Amygos) [reply ui/public/metadata.json:16]: So is the `bug_url` field mandatory?

Comment (edospadoni) [reply ui/public/metadata.json:16]: Yes, otherwise you have a blank line near the "Bugs" field

Comment (Amygos) [reply ui/public/metadata.json:16]: Ok, then the correct value is:
```suggestion
    "bug_url": "https://github.com/NethServer/dev"
```

**Review by Amygos — APPROVED** [review]
